scripts/prettier.js

```diff
@@ -23,9 +23,18 @@ function runPrettier(options) {
   let didWarn = false;
   let didError = false;
 
+  const eslintIgnorePath = path.join(
+    process.env.ROOT_PATH || path.join(__dirname, '..'),
+    '.eslintignore',
+  );
+  const prettierConfigPath = path.join(
+    process.env.ROOT_PATH || path.join(__dirname, '..'),
```

Comment:
We should come up with a better strategy. Environment variables are an escape hatch. How is this script used that requires this change?

Reply:
I could use a global variable instead of an env variable, this would already be a win as it reduces its scope.

The current script assumes too much about where the source of truth for the configuration file is. In the use case in material-ui-x, the configuration files are "higher" (parents) in the file system.

An alternative I can think of is to have a tree traversal logic, that pick the configuration file that is the highest in the file system. I imagine it's what Babel does to find its configuration file. However, this solution seems a bit over-designed for the problem. 

What do you think?

Reply:
> The current script assumes too much about where the source of truth for the configuration file is

Why? 

> I could use a global variable instead of an env variable, this would already be a win as it reduces its scope.

Why can't you use a script argument with `yargs`?

Reply:
> The current script assumes too much about where the source of truth for the configuration file is

Why? 

> I could use a global variable instead of an env variable, this would already be a win as it reduces its scope.

Why can't you use a script argument with `yargs`?

Reply:
Happy to use a solution based on the script argument with yargs